### PR TITLE
Fix horizontal scroll

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Related to #710 by @zackkrida 

## Description

This PR attempts to fix the appearance of horizontal scrolling in the audio and image details pages on mobile. At first I thought the cause was just the logo text in the header but on the audio page the related audios were also causing the page to expand horizontally. I changed the related audios to use the 's' layout on mobile so they hide the waveform and fit in the page.

The current image page still has this problem even on desktop but it is fixed in #682, and the mobile view for the new page is fixed here then.

I also removed some classes from the default layout that were hiding the overflow, I'm not sure if this was necessary in order to the page look good on the iframe?

## Testing Instructions

Run the project and navigate to the audio details page on mobile viewport, see if there is any horizontal scroll. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/styles/vocabulary.scss [hunk 1]
```diff
@@ -18,7 +18,6 @@
 // Components
 @import 'components/button';
 @import 'components/image';
-@import 'components/navbar';
```
Comment (obulat) [inline src/styles/vocabulary.scss:21]: 🔥

File: src/layouts/default.vue [hunk 2]
```diff
@@ -1,12 +1,9 @@
 <template>
-  <div class="app grid h-screen overflow-hidden">
+  <div class="app grid h-screen">
```
Comment (obulat) [inline src/layouts/default.vue:2]: This breaks the separate scrolling of the filters sidebar and the main page. Currently we have this outer `div` full height, and its children have `overflow-y` set to `scroll`.

**Review by obulat — COMMENTED** [review]
I think we need a solution that doesn't set `overflow-hidden` to the outer div, otherwise we break other parts of the page.
Was this PR only supposed to fix Audio pages? Because I still see a scroll bar on image search page.

I love extracting the condition for showing the logo text to a separate computed!

File: src/layouts/default.vue (hunk 2 above)
Comment (krysal) [reply src/layouts/default.vue:2]: I had a feeling there was a purpose to this but I didn't remember the filter sidebar, duh! I think we still can do it with this classes then.

**Comment by krysal** [comment]
@obulat This was supposed to fix the mobile version of both, audio and (new, as the current one will be replaced soon) image pages hiding the text of the logo mainly, but that was before you merged #718 so it was probably already fixed there :) 

I'll update this PR and check it does not break the scrollbar of the filter sidebar and probably will be limited to only change the hide the waveform of related audio on mobile.

**Pull request closed without merging**